src/utils/index.ts

```diff
@@ -6,5 +6,16 @@ import * as PromiseUtils from "./promise.utils";
 import * as URLUtils from "./url.utils";
 import * as WebSocketUtils from "./websocket.utils";
 import * as BrandUtils from "./brand.utils";
+import * as MobileOSDetectAsync from "./mobile-os-detect.utils";
 
-export { ImageUtils, FormatUtils, LocalStorageUtils, ObjectUtils, PromiseUtils, URLUtils, WebSocketUtils, BrandUtils };
+export {
+    ImageUtils,
+    FormatUtils,
+    LocalStorageUtils,
+    ObjectUtils,
+    PromiseUtils,
+    URLUtils,
+    WebSocketUtils,
+    BrandUtils,
+    MobileOSDetectAsync,
```

Comment:
```suggestion
    OSDetectionUtils,
```
Namespace groups the exported function with a functional name. In the client side it will look like this:

```
OSDetectionUtils.mobileOSDetectAsync()
OSDetectionUtils.validateHuaweiCodes()
```

which is more meaningful than

```
MobileOSDetectAsync.mobileOSDetectAsync()
MobileOSDetectAsync.validateHuaweiCodes()
```